Windows/MainWindow.cpp

```diff
@@ -281,6 +283,10 @@ namespace MainWindow
 			NativeMessageReceived("gpu resized", "");
 		}
 
+		if (screenManager) {
+			screenManager->RecreateAllViews();
```

Comment:
Is this the correct thread to do this from?  Shouldn't this happen via an event?

-[Unknown]